app.rb

```diff
@@ -88,7 +88,7 @@ def authorized?
       end
 
       erb :index, locals: {
-        events: decrypted_events&.sort_by { |e| e["iat"] },
+        events: decrypted_events&.sort_by { |e| -e["iat"] },
```

Comment:
another option is `.reverse` (it's another method call so it would be slow with many events, but with a small enough array it's likely fast enough)

```suggestion
        events: decrypted_events&.sort_by { |e| e["iat"] }.reverse,
```